### PR TITLE
week9

### PR DESCRIPTION
**Pull request by raul-2001**

### CODE REVIEW

File: models/item.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        type: String,
+        required: [true, "Please provide item number"]
+    },
+    itemName: {
+        type: String,
+        required: [true, 'Please provide item name'],
+        maxLength: 50,
+    },
+    price: {
+        type: Number,
```
Comment (zjpetersen) [inline models/item.js:15]: Consider adding in some additional validations here.  For example, price should always be positive

File: models/order.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const { required } = require('joi')
+const mongoose = require('mongoose')
+
+const OrderSchema = new mongoose.Schema({
+    item: {
```
Comment (zjpetersen) [inline models/order.js]: I think you want this items array to reference the item schema you have defined in item.js  Try looking at the [subdocuments section](https://mongoosejs.com/docs/subdocs.html) so they reference each other

**Review by zjpetersen — APPROVED** [review]
Approving, but left a few comments for suggestions on improving the schemas

File: models/order.js (hunk 2 above)
Comment (raul-2001) [reply models/order.js]: Mr Zach, thanks for reviewing my code. Yes. It reference the item schema.
I am making an order app, that will contain the items that users will create orders and add the items in it. Just I need a little bit to improve my code.

File: models/item.js (hunk 1 above)
Comment (raul-2001) [reply models/item.js:15]: I added min: 1 , validator. I tested it and it resolves the positive validator issue.

**Pull request merged**